apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/create-page/index.tsx

```diff
@@ -35,22 +36,24 @@ export default function CreatePage( { postType }: Props ) {
 		'a8c.WpcomBlockEditorNavSidebar.createPostLabel',
 		defaultLabel,
 		postType.slug
-	);
+	) as string;
```

Comment:
This forced type asserting feels weird. Is there a better way? Maybe a version of `applyFilter` with generics?

Reply:
> Maybe a version of `applyFilter` with generics?

That's probably the better version — should we start patching it directly in ETK and open a parallel PR in core to improve the typings?

Reply:
That would be better, but is difficult to implement the types for (in a sound way). We just recently got the [hooks package](https://github.com/WordPress/gutenberg/blob/master/packages/hooks/src/index.js) typed and it could likely be improved.